src/plays/smart-todo-list/styles.css

```diff
@@ -0,0 +1,334 @@
+:root {
```

Comment:
Do not override the `:root` as it will conflict the base style of the app. You need to scope these variables for your play.

Reply:
Fixed